Sources/App/routes.swift

```diff
@@ -11,8 +11,29 @@ public func routes(_ router: Router) throws {
     router.get("hello") { req in
         return "Hello, world!"
     }
+    
+    router.post(GithubPullRequestWebhook.self, at: "pullRequestActivity") { (req, webhook) -> Future<Response> in
```

Comment:
URLs are typically hyphenated. This should be `pull-request-activity` or `pull-request`. 

Side note, do we want performance tests to run when a PR is open? What if it's just fixing a typo in comments? Or it's an invalid / not ready to merge PR? I think it makes sense to ask the bot each time you want to run it.

Reply:
For the URL, maybe `/github/pr` actually. That way `/circle` and `/github` are separate, "namespaced" endpoints.

Reply:
Good point on the namespacing. I'll do that.

Perf tests won't be run upon opening PRs. Only merging them. Otherwise they'll have to be triggered by the bot.

```swift
// This route will only be responding to merges
guard
    webhook.action == "closed",
    webhook.pullRequest.merged == true
else {
    return try HTTPStatus.ok.encode(for: req)
}
```

Reply:
routes are namespaced. Relies on https://github.com/vapor/routing/pull/68